backend/baseclass.pm

```diff
@@ -908,13 +906,13 @@ sub _reduce_to_biggest_changes {
 
 sub freeze_vm {
     my ($self) = @_;
-    bmwqemu::diag "ignored freeze_vm";
+    bmwqemu::fcterr("ignored freeze_vm");
```

Comment:
IMHO that's not an error, it's info. The user should be informed if this is called on a backend that does not support it but it can be expected behaviour